### DISCUSSION TITLE
New language for qualifier: nix

### DISCUSSION THREAD

**Discussion by MichaelHinrichs** [community/community#123023] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Here's a repo that uses nix. https://github.com/urbit/urbit

**Comment by LiteBrite82** [comment]
@MichaelHinrichs 
Thanks for highlighting new tech! Can you elaborate a little more on your post? Is this related to GitHub Code Search, or, is this just general information? 

(Its great community participation either way, I'm just looking for context so it can be categorized appropriately)

**Reply by MichaelHinrichs** [reply]
Yeah, i would like it to be added to the searchable languages.
![Screenshot 2024-05-10 104819](https://github.com/community/community/assets/37460517/ba9d2647-e0a2-49b2-baef-59e759ce2741)

**Comment by look** [comment, marked as answer]
Nix is searchable: https://github.com/search?q=repo%3Aurbit%2Furbit%20lang%3ANix&type=code

**Reply by MichaelHinrichs** [reply]
Thank you!
